shared/runtime/pyexec.c

```diff
@@ -583,7 +583,7 @@ int pyexec_friendly_repl(void) {
     for (;;) {
     input_restart:
 
-        #if MICROPY_HW_ENABLE_USB
+        #if MICROPY_HW_ENABLE_USB && 0
         if (usb_vcp_is_enabled()) {
             // If the user gets to here and interrupts are disabled then
             // they'll never see the prompt, traceback etc. The USB REPL needs
```

Comment:
@dpgeorge This block in the shared pyexec code appears to be stm32 specific! Other ports use the `MICROPY_HW_ENABLE_USBDEV` define and don't define `usb_vcp_is_enabled` etc. 
This little feature to re-enable any disabled interrupts once a device hits repl does sound convenient, would it be worth extending this to all ports? Or does this feel like something users haven't needed and should be removed?

Reply:
I reckon we can just remove this.  It's pretty hacky and doesn't generalise well to other ports.